notebooks/113-image-classification-quantization/113-image-classification-quantization.ipynb

```diff
@@ -0,0 +1,446 @@
+        "from compression.graph.model_utils import compress_model_weights\n",
+        "from compression.engines.ie_engine import IEEngine\n",
+        "from compression.pipeline.initializer import create_pipeline\n",
+        "from openvino.inference_engine import IECore\n",
+        "\n",
+        "sys.path.append('../utils')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Set the data and model directories\n",
+        "DATA_DIR = 'data'\n",
+        "MODEL_DIR = 'model'\n",
+        "\n",
+        "os.makedirs(DATA_DIR, exist_ok=True)\n",
+        "os.makedirs(MODEL_DIR, exist_ok=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "D-frbVLKrkmv"
+      },
+      "source": [
+        "## Prepare the Model\n",
+        "Model preparation stage has the following steps:\n",
+        "- Download PyTorch model from Torchvision repository\n",
+        "- Convert it to ONNX format\n",
+        "- Run OpenVINO Model Optimizer tool to convert ONNX to OpenVINO Intermediate Representation (IR)\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "f7i6dWUmhloy"
+      },
+      "outputs": [],
+      "source": [
+        "model = torch.hub.load(\"chenyaofo/pytorch-cifar-models\", \"cifar10_mobilenetv2_x1_0\", pretrained=True)\n",
+        "model.eval()\n",
+        "\n",
+        "dummy_input = torch.randn(1, 3, 32, 32)\n",
+        "\n",
+        "onnx_model_path = Path(MODEL_DIR) / 'mobilenet_v2.onnx'\n",
+        "ir_model_xml = onnx_model_path.with_suffix('.xml')\n",
+        "ir_model_bin = onnx_model_path.with_suffix('.bin')\n",
+        "\n",
+        "torch.onnx.export(model, dummy_input, onnx_model_path, verbose=True)\n",
+        "\n",
+        "# Run OpenVINO Model Optimization tool to convert ONNX to OpenVINO IR\n",
+        "!mo --framework=onnx --data_type=FP16 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ynLvh8rNc2wv"
+      },
+      "source": [
+        "## Define Data Loader\n",
+        "At this step the `DataLoader` interface from POT API is implemented."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.4914, 0.4822, 0.4465), (0.247, 0.243, 0.261))])\n",
+        "dataset = CIFAR10(root=DATA_DIR, train=False, transform=transform, download=True)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "DErQofk8tO6c"
+      },
+      "outputs": [],
+      "source": [
+        "class CifarDataLoader(DataLoader):\n",
+        "\n",
```

Comment:
I suggest using the docstring format. It provides convenient associate documentation for modules, functions, classes, and methods. Also, I recommend including type hints on it. It is super important that the developer (reader of the notebook) understand each step. See the attached image with docstring and type hints examples:

![image](https://user-images.githubusercontent.com/10940214/153489454-840292e0-2168-4a9d-88ca-dcc8223b50a4.png)

Please check all functions and classes in this notebook.